CHANGELOG.md

```diff
@@ -49,8 +49,9 @@
   - `artist` / `title` / `track_alt`: fix several artists and titles which had pieces
     incorrectly identified as `track_alt`.
 
-- search: remove character **&** from the search query which Bandcamp often fails to
-  handle
+- `search`:
+  - properly escape query strings for better results with special characters
+  - change HTTP client implementation to avoid Bandcamp "403 Forbidden" responses
```

Comment:
I updated the v0.20.0 changelog because AFAICT it's not actually released yet!

Let me know if I should create a new section, or do anything else.

Reply:
I see this is now updated with a new patch version - thanks. And especially thanks for following the changelog format I've been using!

Feel free to use today's date in **2024-??-??** - I think we'll be able to merge it today.

Also, it would be good to update the package version in `pyproject.toml` accordingly.